o8g/Scripts/actions.py

```diff
@@ -1619,6 +1873,20 @@ def search(group, count = None):
         moved += 1
     me.piles['Discard Pile'].lookAt(moved)
     
+def searchTopDeck(group, target, count = None):
```

Comment:
Since you set a default value for count, it would be nice if you would automatically take the length of group as count when count is omitted from the function call.